test/e2e/operator/nmstate_install_test.go

```diff
@@ -0,0 +1,110 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	nmstatev1beta1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1beta1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	v1 "k8s.io/api/apps/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	nmstate     nmstatev1beta1.NMState
+	nmstateKey  = types.NamespacedName{Namespace: "nmstate", Name: "nmstate"}
+	handlerKey  = types.NamespacedName{Namespace: "nmstate", Name: "nmstate-handler"}
+	operatorKey = types.NamespacedName{Namespace: "nmstate", Name: "nmstate-operator"}
+	webhookKey  = types.NamespacedName{Namespace: "nmstate", Name: "nmstate-webhook"}
+)
+
+func installNMState() {
+	err := framework.Global.Client.Get(context.TODO(), nmstateKey, &nmstate)
+	Expect(err).To(HaveOccurred())
+	err = framework.Global.Client.Create(context.TODO(), &nmstate, &framework.CleanupOptions{})
+	Expect(err).To(Not(HaveOccurred()))
+}
+
+func uninstallNMState() {
+	err := framework.Global.Client.Get(context.TODO(), nmstateKey, &nmstate)
+	Expect(err).To(Not(HaveOccurred()))
+	err = framework.Global.Client.Delete(context.TODO(), &nmstate, &client.DeleteOptions{})
+	Expect(err).To(Not(HaveOccurred()))
+}
+
+var _ = Describe("NMState operator", func() {
+	BeforeEach(func() {
+		nmstate = nmstatev1beta1.NMState{
```

Comment:
Please initialize the variable as a part of the Describe context. To keep it visible and reduce the scope.